soccer_vision_msgs/msg/MarkingIntersection.msg

```diff
@@ -0,0 +1,29 @@
+# Describes a detected marking intersection from vision modules, in image coordinates.
+# An intersection is defined as a point where two or more line markings intersect.
+# The term ray is used to describe the outgoing rays from the center of the
+# intersection.
+#
+# For example, looking down at a perfectly upright T-intersection on the ground from above, you would have:
+# - num_rays = 3
+# - heading_rays = [0, pi / 2, pi]
+
+float32 CONFIDENCE_UNKNOWN=-1
+
+# Center point of the intersection, the z-axis should be ignored (in pixel)
+geometry_msgs/Point center
+
+# The number of rays outgoing from an intersection. This is 3 for a T-Junction and 
+# 4 for an X-Junction
+int32 num_rays
+
+# NOTE: The length of this array should match num_rays.
```

Comment:
I just discussed with @SammyRamone the case, that we only know (or detect) the "number of rays", as the Bit-Bots only detect the intersection class (one of L-, T- or X-Intersection).
Thereby we don't know the heading of the rays, which would also be rather complex to integrate.

We would prefer a solution, where we set the `num_rays` variable (inferred from the detected class) but leave the array of ray headings empty (or n times `-1`).

Reply:
That is a case that is not well considered with the existing msg. May I suggest that we update the comment for this field to handle your case? How about

```
# The length of the array should either:
# - Match the number of rays specified with num_rays OR
# - Be empty, if the headings are unknown
# Heading is defined as 0 along the +x axis, and pi / 2 along the +y axis.
# Headings should:
# - have an interval of [0, 2pi)
# - be stored in ascending order

```